### ISSUE TITLE
New User, Questions, comments and concerns

### ISSUE THREAD

**Issue by samlovesgithub** [origin0110/OriginShader#9]
Hello there Developers and users of this lovely shader. I have a few things to say if you don't mind. 

First of all, fantastic job! The shader is gorgeous! The tone is even, the saturation is perfect and the exposure is lovely! 

However the performance is awful. Can you please figure out a way to reduce the weight of the shader on my gpu? I have an iPad iOS 7th gen, 2-3gb of ram. It's strength is the water animation and style but frankly it impacts performance greatly. Have a great day!

**Comment by paula867** [comment]
Yep, i also have this issue with the iPad. It's a wonderful shader, I would love to open a world with it but i don't have a powerful enough device. :(

**Comment by saintdubu** [comment]
when will it be available for windows 10?

**Comment by origin0110** [comment]
Part of the optimization has been completed. I think it should be lighter than before :)

**Comment by samlovesgithub** [comment]
> Part of the optimization has been completed. I think it should be lighter than before :)

Have you released a lighter and more optimized version of the shader then? Please say you have!

**Comment by origin0110** [comment]
Yes, the sky rendering performance of versions after 3.4.3 is about 50% better than that of 3.3.x. Although it is still too laggy for low 
 devices.

**Comment by Scriptov** [comment]
I just can’t download the shader. Can you know what is the reason?
